test/interactiveutil.jl

```diff
@@ -0,0 +1,27 @@
+
+file, line = functionloc(workspace)
+func, str = editcmd(file, line)
+@test func == run || func == spawn
+@test contains(string(str), file)
+
+
+@unix_only begin
+    try
+        clipboard("Clipboard Test")
```

Comment:
is it worth saving the existing contents first and restoring them in a `finally` block?